bedrock/privacy/templates/privacy/notices/base-notice.html

```diff
@@ -13,6 +13,7 @@
 {% block body_class %}{{ super() }} format-none{% endblock %}
 
 {% block title %}{{ doc.h1.string|join|safe }}{% endblock%}
+{% block page_desc %}{{ _('Know what data Firefox shares and how we receive, handle and share information we collect from Firefox.') }}{% endblock %}
```

Comment:
Please make this change in `bedrock/privacy/templates/privacy/base-protocol.html` and use the following string:

`Our Privacy Notices describe the data our products and services receive, share, and use, as well as choices available to you.`

Thanks!